lighthouse-core/gather/gatherers/script-elements-snapshot.js

```diff
@@ -76,15 +82,14 @@ class ScriptElementsSnapshot extends FRGatherer {
   async stopSensitiveInstrumentation(context) {
     const session = context.driver.defaultSession;
     await session.sendCommand('Debugger.disable');
-    await session.off('Debugger.scriptParsed', this.onScriptParsed);
+    if (this.onScriptParsed) session.off('Debugger.scriptParsed', this.onScriptParsed);
```

Comment:
```suggestion
    if (this.onScriptParsed) await session.off('Debugger.scriptParsed', this.onScriptParsed);
```

Reply:
`session.off` and `session.on` return `void` no?